modin/pandas/io.py

```diff
@@ -825,7 +825,7 @@ def read_orc(
 
 
 @_inherit_docstrings(pandas.HDFStore)
-class HDFStore(ClassLogger, pandas.HDFStore):  # noqa: PR01, D200
+class HDFStore(ClassLogger, pandas.HDFStore, modin_layer="PANDAS-IO"):  # noqa: PR01, D200
```

Comment:
```suggestion
class HDFStore(ClassLogger, pandas.HDFStore):  # noqa: PR01, D200
```
This is also pandas API.